src/Playroom/SettingsPanel/SettingsPanel.tsx

```diff
@@ -73,101 +74,154 @@ const KeyboardShortcut = ({
   );
 };
 
+const getTitle = (localTitle: string | undefined) => {
+  if (localTitle) {
+    return `${localTitle} | Playroom`;
+  }
+
+  const configTitle = window?.__playroomConfig__.title;
+
+  if (configTitle) {
+    return `${configTitle} | Playroom`;
+  }
+
+  return 'Playroom';
+};
+
 export default React.memo(() => {
-  const [{ editorPosition, colorScheme }, dispatch] = useContext(StoreContext);
+  const [{ editorPosition, colorScheme, localTitle }, dispatch] =
+    useContext(StoreContext);
 
   const keybindings = getKeyBindings();
 
+  const displayedTitle = getTitle(localTitle);
+
   return (
-    <ToolbarPanel data-testid="frame-panel">
-      <Stack space="large" dividers>
-        <fieldset className={styles.fieldset}>
-          <legend>
-            <Heading level="3">Editor Position</Heading>
-          </legend>
-          <div className={styles.radioContainer}>
-            {['Bottom', 'Right'].map((option) => (
-              <div key={option}>
-                <input
-                  type="radio"
-                  name="editorPosition"
-                  id={`editorPosition${option}`}
-                  value={option.toLowerCase()}
-                  title={option}
-                  checked={option.toLowerCase() === editorPosition}
-                  onChange={() =>
-                    dispatch({
-                      type: 'updateEditorPosition',
-                      payload: {
-                        position: option.toLowerCase() as EditorPosition,
-                      },
-                    })
-                  }
-                  className={styles.realRadio}
-                />
-                <label
-                  htmlFor={`editorPosition${option}`}
-                  className={styles.label}
-                  title={option}
-                >
-                  <span className={styles.labelText}>
-                    {positionIcon[option.toLowerCase() as EditorPosition]}
-                  </span>
-                </label>
+    <>
+      {localTitle === undefined ? null : (
+        <Helmet>
+          <title>{displayedTitle}</title>
+        </Helmet>
+      )}
+      <ToolbarPanel data-testid="frame-panel">
+        <Stack space="small" dividers>
```

Comment:
The divider looks a bit close, this should possibly be large like [here] but we can play with this when we update the field (as below).

[here]: https://github.com/seek-oss/playroom/pull/319/files#diff-4021ca4ad6a2db46421d46681ae15da803ba3543b1cee9ba7100027136eca0c8R135